temporian/core/event_set_ops.py

```diff
@@ -4382,3 +4382,46 @@ def drop(
         from temporian.core.operators.select import drop
 
         return drop(self, feature_names=feature_names)
+
+    def assign(
+        self: EventSetOrNode, **others: EventSetOrNode
+    ) -> EventSetOrNode:
+        """Assign new features to an EventSet
+
+        If the name provided already exists on the EventSet, the feature is
+        overriden.
```

Comment:
If the feature names provided already exist on the EventSet, those features are replaced by the new ones?

Reply:
Yes! I replicated that functionality from pandas.

Inside the assign, I first drop the existing ones otherwise the glue would fail